### PR TITLE
Allow to ignore NFTS in total netvalue

### LINKED ISSUE #3549: Option to not count NFTs in net worth
It would be useful to have a toggle to not count NFTs in the total net worth even if the module is activated or a toggle in the dashboard to not display them in graphs and calculations

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3549 

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/db/dbhandler.py [hunk 1]
```diff
@@ -2660,21 +2661,33 @@ def get_rotkehlchen_premium(self) -> Optional[PremiumCredentials]:
...
         )
-
+        historical_data = query.fetchall()
+        if not include_nfts:
+            nft_balances = cursor.execute(
+                f'SELECT time, SUM(usd_value) FROM timed_balances WHERE time >= ? '
+                f'AND currency LIKE "{NFT_DIRECTIVE}%" GROUP BY time',
+                (from_ts,),
+            )
+        nft_values = {time: value for time, value in nft_balances}
```
Comment (LefterisJP) [inline rotkehlchen/db/dbhandler.py:2684]: This is not wrong and will work. Just thinking a bit here. Can we optimize a bit by changing the logic up?
Idea.

Remove `nft_balances` from 2671 and make:

```suggestion
        historical_data = query.fetchall()
        if not include_nfts:
            nft_balances = cursor.execute(
                f'SELECT time, SUM(usd_value) FROM timed_balances WHERE time >= ? '
                f'AND currency LIKE "{NFT_DIRECTIVE}%" GROUP BY time',
                (from_ts,),
            )
            nft_values = {time: value for time, value in nft_balances}
```

And then in 2687:

```python
for entry in historical_data:
    times_int.append(entry[0])
    if include_nfts:
        data.append(entry[1])
    else:
        data.append(FVal(entry[1]) - FVal(nft_values.get(entry[0], 0)))
```

This way the code is faster for the most common case, which is to include the options for NFTs and probably the other too.

1. `nft_values` is only created in the case where it's needed
2. in every iteration of the historical data loop there is an extra if check, but that removes quite a bit of extra logic from  the most common case which I asume would be the current behaviour of taking nft into consideration.

File: rotkehlchen/tests/utils/rotkehlchen.py [hunk 2]
```diff
@@ -313,3 +313,60 @@ def add_starting_balances(datahandler) -> List[DBAssetBalance]:
     datahandler.db.add_multiple_location_data(location_data)
 
     return balances
+
+
+def add_starting_nfts(datahandler):
+    """Adds a time serie for an account owning a NFT"""
```
Comment (LefterisJP) [inline rotkehlchen/tests/utils/rotkehlchen.py:319]:
```suggestion
    """Adds a time series for an account owning a NFT"""
```

**Review by LefterisJP — COMMENTED** [review]
Left some comments

**Review by kelsos — COMMENTED** [review]
Frontend changes look good to me :+1:

File: rotkehlchen/db/dbhandler.py [hunk 3]
```diff
@@ -2660,21 +2661,34 @@ def get_rotkehlchen_premium(self) -> Optional[PremiumCredentials]:
...
+            nft_balances = cursor.execute(
+                f'SELECT time, SUM(usd_value) FROM timed_balances WHERE time >= ? '
+                f'AND currency LIKE "{NFT_DIRECTIVE}%" GROUP BY time',
+                (from_ts,),
+            )
+            nft_values = {time: value for time, value in nft_balances}
         data = []
         times_int = []
-        for entry in query:
+        for entry in historical_data:
```
Comment (LefterisJP) [inline rotkehlchen/db/dbhandler.py:2686]: Here you are also looping the `historical_data` 2 times.

1. in the fetchall()
2. in the loop

Just remove  `historical_data = query.fetchall()` and replace `for entry in historical_data` with `for entry in query`

**Review by LefterisJP — APPROVED** [review]
LGTM. Let's merge this. Both e2e and backend tests hit known failures.

**Pull request merged**